### PR TITLE
AMQ: multiple hosts in AmqpDetailsConnectionProvider

### PR DESCRIPTION
**Pull request by EldarKZ16**
## Purpose
Create AmqpDetailsConnectionProvider with multiple hosts and ports

## References

References #xxxx

### CODE REVIEW

**Comment by lightbend-cla-validator** [comment]
Hi @EldarKZ16,

Thank you for your contribution! We really value the time you've taken to put this together.

Before we proceed with reviewing this pull request, please sign the Lightbend Contributors License Agreement:

[http://www.lightbend.com/contribute/cla](http://www.lightbend.com/contribute/cla)

File: amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnectionProvider.scala [hunk 1]
```diff
@@ -203,6 +203,9 @@ object AmqpDetailsConnectionProvider {
   def apply(host: String, port: Int): AmqpDetailsConnectionProvider =
     new AmqpDetailsConnectionProvider(immutable.Seq(host -> port))
 
+  def apply(hostAndPorts: immutable.Seq[(String, Int)]): AmqpDetailsConnectionProvider =
```
Comment (ennru) [inline amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnectionProvider.scala]: Please add it to the Java API with `j.u.Collection<akka.japi.Pair<String, Integer>>`, as well.

Comment (EldarKZ16) [reply amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnectionProvider.scala]: Done

Comment (ennru) [reply amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnectionProvider.scala]: You need to add some conversion logic...

Comment (EldarKZ16) [reply amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnectionProvider.scala]: Oh, I forgot, sorry about this. Fixed.

Comment (ennru) [reply amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnectionProvider.scala]: Oops, your change is adapting the wrong data.

Comment (EldarKZ16) [reply amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnectionProvider.scala]: I'm Junior Backend developer :)

**Comment by ennru** [comment]
Thank you! Keep the improvements coming.

**Pull request merged**